pkgs/tools/misc/gay/default.nix

```diff
@@ -13,6 +13,12 @@ python3Packages.buildPythonApplication rec {
     sha256 = "sha256-vouEFybcz27bcw/CpAGjFY8NYWQC+V0IE7h1a8XufZ0=";
   };
 
+  preBuild = ''
```

Comment:
Maybe it makes more sense to do this in postUnpack?

Reply:
Also maybe explain why/how this fixes the build?

Reply:
I find the better way to solve this (i.e. without _weirdly_ removing directories).
But the problem was gay's setup.py does not exclude docker/ and preview/ directory 
so setuptools *thinks* they are one of python packages despite the actual fact it's not.